test/uk/gov/hmrc/platforminitiatives/services/PlatformInitiativesServiceSpec.scala

```diff
@@ -0,0 +1,170 @@
+/*
+ * Copyright 2021 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package uk.gov.hmrc.platforminitiatives.services
+
+import org.mockito.MockitoSugar
+import org.mockito.ArgumentMatchers.any
+import org.scalatest.concurrent.ScalaFutures
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpec
+import play.api.mvc.ControllerComponents
+import uk.gov.hmrc.platforminitiatives.connectors.{RepositoryDisplayDetails, ServiceDependenciesConnector, TeamsAndRepositoriesConnector}
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.platforminitiatives.models.{Dependencies, Dependency, PlatformInitiative, Version}
+
+import java.time.LocalDateTime
+import scala.concurrent.ExecutionContext.Implicits.global
+import scala.concurrent.duration.DurationInt
+import scala.concurrent.{Await, Future}
+import scala.language.postfixOps
+
+class PlatformInitiativesServiceSpec extends AnyWordSpec with Matchers with MockitoSugar with ScalaFutures {
+
+  "createDefaultBranchInitiative" should {
+    "return an initiative for DefaultBranches where branch name is not updated" in new Setup {
+      when(mockTeamsAndRepositoriesConnector.allDefaultBranches(any[HeaderCarrier])) thenReturn
+        Future.successful(mockRepositories)
+      val result: Future[PlatformInitiative] = platformInitiativesService.createDefaultBranchInitiative(
+        initiativeName        = "Test",
+        initiativeDescription = "Test Description",
+        completedLegend       = "Updated",
+        inProgressLegend      = "Master"
+      )
+      val finalResult: PlatformInitiative = Await.result(result, 1 second)
```

Comment:
you already have `ScalaFutures` mixed in - you can replace the `Await` with `result.futureValue`. If the default timeout is not enough, you can also mix in `IntegrationPatience`